iron-list.html

```diff
@@ -642,18 +641,13 @@
           this._lastVisibleIndexVal = lastIndex > this._virtualCount ? this._virtualCount : lastIndex;
         } else {
           var physicalOffset = this._physicalTop;
+          this._lastVisibleIndexVal = this._virtualStart;
 
           this._iterateItems(function(pidx, vidx) {

```

Comment:
Seems like you could break out of `_iterateItems` as soon as `physicalOffset > this._scrollBotton`, similar to `get firstVisibleIndex()`.